src/styles/blocks/header.scss

```diff
@@ -0,0 +1,88 @@
+.header {
+  background-size: cover;
+  background-position: top right 15%;
+  background-image: url(../images/Cover_photo.jpg);
+
```

Comment:
the header and footer should also be limited in width so that the site does not look strange on large screens

![footer](https://user-images.githubusercontent.com/89162624/209549004-56189ad1-b0e5-49cc-b5eb-11cd8f690e10.png)
![header](https://user-images.githubusercontent.com/89162624/209549010-32eb7e65-f89f-44e9-a35e-09c83ad64660.png)

Reply:
i fixed it